src/CodeGen_RISCV.cpp

```diff
@@ -50,7 +51,7 @@ string CodeGen_RISCV::mattrs() const {
     string arch_flags = "+m,+a,+f,+d,+c";
 
     if (target.has_feature(Target::RVV)) {
-        arch_flags += ",+experimental-v";
+        arch_flags += ",+v";
```

Comment:
Does this need to be conditional on LLVM_VERSION?

Reply:
Interesting question. I'm tempted to say we don't want to support something that was experimental. Like if one is using RISC V vector, one wants to be at least this recent on LLVM. (And it will not be a build time issue.) Note it's been broken for maybe a few months so anyone actually using it was making this change by hand. Think it is a safe bet to not conditionalize.